activesupport/lib/active_support/cache/strategy/local_cache.rb

```diff
@@ -116,6 +116,19 @@ def read_entry(key, options)
             end
           end
 
+          def read_multi_entries(names, options)
+            return super unless local_cache
+
+            local_entries = Hash[names.map { |name| [name, local_cache.read_entry(name, options)] }.keep_if { |_name, value| value }]
```

Comment:
Let's extract to `LocalStore#read_multi_entries`

Reply:
Extract this line to another method?

This is the name of this method itself.